### ISSUE TITLE
xiaomi-mi-13-i5-8250 in 2022

### ISSUE THREAD

**Issue by webcaptcha** [Golovin-Andrey/xiaomi-mi-13-ubuntu#12]
Do you still using this Laptop? What is the best way to configure graphics in Linux?

**Comment by Golovin-Andrey** [comment]
Yes, I do. Today I have only half of battery capacity and don't use nvidia card

**Comment by webcaptcha** [comment]
Did you blacklist nvidia card? Because in the repo I see settings to configure bumblee. I just have the same laptop. Appreciate if you share config.
